graphix/open_graph.py

```diff
@@ -0,0 +1,383 @@
+"""Provides a class for open graphs."""
+
+from __future__ import annotations
+
+from copy import deepcopy
+from dataclasses import dataclass
+
+import networkx as nx
+import numpy as np
+import pyzx as zx
+
+
+@dataclass
+class Measurement:
```

Comment:
why not use `graphix.command.M`? we will soon have Pauli measurement recognition under development: https://github.com/TeamGraphix/graphix/pull/189

Reply:
I have considered it, however I felt that it wasn't a good separation of concerns. From what I understand, `graphix.command.M` is a command defined at the pattern level and hence has methods and attributes useful for this layer of abstraction e.g. correction parameters.
Whereas `Measurement` is more like a tuple (angle, plane) with methods useful for a higher level of abstraction at the level of Open graphs.

Though I'm happy to discuss it more

Reply:
Indeed they have different purposes. I think I agree.